redash/models/__init__.py

```diff
@@ -304,22 +322,12 @@ def store_result(cls, org, data_source, query_hash, query, data, run_time, retri
                            data_source=data_source,
                            retrieved_at=retrieved_at,
                            data=data)
+        
+        
         db.session.add(query_result)
         logging.info("Inserted query (%s) data; id=%s", query_hash, query_result.id)
-        # TODO: Investigate how big an impact this select-before-update makes.
-        queries = Query.query.filter(
-            Query.query_hash == query_hash,
-            Query.data_source == data_source
-        )
-        for q in queries:
-            q.latest_query_data = query_result
-            # don't auto-update the updated_at timestamp
-            q.skip_updated_at = True
-            db.session.add(q)
-        query_ids = [q.id for q in queries]
-        logging.info("Updated %s queries with result (%s).", len(query_ids), query_hash)
```

Comment:
We moved this to a separate class method on the `Query` model. Eventually wasn't needed for this refactor, but it's a good separation regardless so we kept it here.